tests/dist/python/run_dist_objects.py

```diff
@@ -31,6 +32,70 @@ def run_server(graph_name, server_id, server_count, num_clients, shared_mem, kee
     print('start server', server_id)
     g.start()
 
+##########################################
+############### DistGraph ###############
+##########################################
+
+def dist_graph_test(g):
```

Comment:
As `DistGraph` is the key class in `DistDGL`, we'd better test all the public `methods/attributes`. Could you list all of them and test one by one? I think some are missing such as `node_attr_schemes`, `in_degrees` and so on.